lib/roo/base.rb

```diff
@@ -32,6 +32,14 @@ def initialize(filename, options = {}, _file_warning = :error, _tmpdir = nil)
     @last_column = {}
 
     @header_line = 1
+  rescue => e
```

Comment:
Should it be `rescue` or `finally`?
Using `rescue` means this only "closed" when exception is caught